engine/token_rotation.tf

```
@@ -250,7 +250,7 @@ EOF
 }
 
 resource "aws_cloudwatch_log_group" "rotate_token_state_machine" {
-  name = "ServiceCatalogTerraformCloudTokenRotationStateMachine"
+  name = "ServiceCatalogTerraformCloudTokenRotationSSM"
```

Suggestion: Should we remove "Cloud" from the name here as well? Also, it looks like an extra "S" might have snuck into this name ✂️ 
```suggestion
  name = "ServiceCatalogTerraformTokenRotationSM"
```